### DISCUSSION TITLE
Ability to make subclaims that block people with permission in a claim from interacting

### DISCUSSION THREAD

**Discussion by Hedwig7s** [GriefPrevention/GriefPrevention#1857] [Ideas]
I have a friend who likes to take everything without asking. I want to be able to block them from my chests without having to reformat my claim or only allow them on parts of my claim. This would be useful for that

**Comment by Jikoo** [comment]
`/RestrictSubclaim` already exists.

**Comment by xSepticSidx** [comment]
Late I know but in case anyone else reads it.
If you allow trust on the claim a friend can build.

Create a subdivision around the chest area, use the above in the config (so you may have to ask the server owner about this) so that
your friend loses permission to open the chests but you still can.

You can then give containertrust to others that you do trust.

**Reply by Jikoo** [reply]
It's not a configuration option, just a command for a claim setting. I assume you mean "when configuring the claim," but you do not need admin action to be able to do this.
